tket2/src/serialize/pytket/op/serialised.rs

```diff
@@ -91,13 +88,13 @@ impl OpaqueTk1Op {
         if custom_op.name() != format!("{TKET1_EXTENSION_ID}.{TKET1_OP_NAME}") {
             return Ok(None);
         }
-        let Some(TypeArg::Opaque { arg }) = custom_op.args().first() else {
+        let Some(TypeArg::String { arg }) = custom_op.args().first() else {
             return Err(serde_yaml::Error::custom(
                 "Opaque TKET1 operation did not have a yaml-encoded type argument.",
             )
             .into());
         };
-        let op = serde_yaml::from_value(arg.value.clone())?;
+        let op = serde_yaml::from_str(arg)?;
```

Comment:
```suggestion
        let op = serde_json::from_str(arg)?;
```

You will also have to replace `_yaml` by `_json` in the custom error a couple lines above, and `serde_yaml::Error` with `serde_json::Error` in the `OpConvertError` definition on `serialize/pytket.rs:213`

Reply:
After that we should be able to drop the `serde_yaml` dependency.